go-controller/pkg/node/healthcheck.go

```diff
@@ -158,6 +159,11 @@ func checkForStaleOVSInternalPorts() {
 	staleInterfaceArgs := []string{}
 	values := strings.Split(stdout, "\n\n")
 	for _, val := range values {
+		if val == types.K8sMgmtIntfName || val == types.K8sMgmtIntfName+"_0" {
```

Comment:
If I understand correctly, only the rep (types.K8sMgmtIntfName_0) is plugged to br-int bridge and has a ofport number, why checking types.K8sMgmtIntfName for stale ovs port?

Reply:
Because after reboot all the reps will be recreated with, probably, different ifindex. So ovs port for `ovn-k8s-mp0` became stale port.

Reply:
I thought we only add VF rep to OVS integration bridge, not the VF netdev, how would the VF netdev become stale OVS port?

Reply:
You're correct, we only add VF rep to OVS bridge (and SF rep in the future). Let me clarify a bit. `types.K8sMgmtIntfName` value is `ovn-k8s-mp0`. This value will be given to representor when mode is DPU, and to netdevice when mode is DPU host. In the full mode both representor and netdevice presented on the same host, so netdevice will be given name `ovn-k8s-mp0` and representor `ovn-k8s-mp0_0`. So this check verifies if port have name `types.K8sMgmtIntfName`, which is for DPU/DPU-host mode or port have name `types.K8sMgmtIntfName+ "_0"` which is for full mode. In both cases lookup will be for representors only.

Reply:
I see, I missed the DPU/DPU-host mode, only thinking about FULL mode.